parachain/pallets/ethereum-beacon-client/src/weights.rs

```diff
@@ -1,47 +1,34 @@
-//! Autogenerated weights for ethereum_beacon_client
+//! Autogenerated weights for `snowbridge_ethereum_beacon_client`
```

Comment:
In the long term this file belongs in the cumulus repo and is generated by their benchmarking scripts on the hardware they specified for bridge hub.